frontend/src/components/RideModal/Pages/RiderInfo.tsx

```diff
@@ -91,7 +83,7 @@ const RiderInfoPage = ({ formData, onBack, onSubmit }: ModalPageProps) => {
           )}
           <datalist id="locations">
             {locations.map((l) => (
-              <option key={l}>{l}</option>
+              <option key={l.id}>{l.id}</option>
```

Comment:
should be `l.name`, or else this happens
![image](https://user-images.githubusercontent.com/20343975/144279855-a26e6074-75fa-42d9-b542-c5f9d477d015.png)